### ISSUE TITLE
"import vtk" fails after update to version 9

### ISSUE THREAD

**Issue by 2manek** [msys2/MINGW-packages#14375]
### Description

after update vtk from version 8.2.0 to 9.2.2 I can not import vtk

```
>>> import vtk
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "E:/msys64.new/mingw64/lib/python3.10/site-packages/vtk.py", line 31, in <module>
    all_m = importlib.import_module('vtkmodules.all')
  File "E:/msys64.new/mingw64/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "E:/msys64.new/mingw64/lib/python3.10/site-packages/vtkmodules/all.py", line 7, in <module>
    from .vtkCommonCore import *
ModuleNotFoundError: No module named 'vtkmodules.vtkCommonCore'
```

```
$ python --version
Python 3.10.8
```

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

Microsoft Windows [Version 10.0.25252.1010]

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [ ] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

### Expected behavior

import vtk with no errors

### Actual behavior

vtk modules not found

### Repro steps

1. download current version of msys2
2. update msys2 via pacman
3. install mingw-w64-x86_64-python
4. install mingw-w64-x86_64-vtk
6. run `python -c "import vtk"`

**Comment by MehdiChinoune** [comment]
Is `python-matplotlib` installed, if yes then try installing optional dependencies one by one and see which one fixes the issue.

**Comment by 2manek** [comment]
I already have all optional vtk dependencies installed. I can't import vtk both when matplotlib is not installed and when it is installed

**Comment by brisingraerowing** [comment]
The VTK 9.2.2 python libraries have ',cpython310' bit, whereas the 8.x ones don't have anything of that sort. Try removing that from the file names and see if it imports. I have a feeling that's what's messing Python up.

**Comment by 2manek** [comment]
OK, I see that vtkExtractEdges has been moved from vtkFiltersExtraction to vtkFiltersCore

**Comment by brisingraerowing** [comment]
Good to know that it worked. That needs to be fixed.

**Comment by 2manek** [comment]
Yes, but on Linux it work fine, at least on my Manjaro with VTK 9.1.0

**Comment by 2manek** [comment]
I think it should be called *.cp310-mingw_x86_64.pyd, but I haven't checked that yet.

**Comment by brisingraerowing** [comment]
The Arch libs have the system triplet as well, and the MinGW ones don't. That being missing is probably throwing Python off.

**Comment by MehdiChinoune** [comment]
I think I have found the issue. VTK uses CMake `Python3_SOABI` as a default suffix, but CMake uses `import sysconfig; sysconfig.get_config_var('SOABI')` to get which gives a wrong value.

I think this issues should be fixed in mingw-w64-python

**Comment by MehdiChinoune** [comment]
@2manek Would you like to test packages built in #14476 (Checks -> arctifacts (right-corner)).

**Comment by MehdiChinoune** [comment]
Fixed by #14478
